ansible/environments.yml

```diff
@@ -17,6 +17,8 @@
 #| m1.xlarge | 16384 |   40 |     8 |
 #+-----------+-------+------+-------+
 
+# Using Image ID d324da4b-b827-44fb-8331-bfad41a28aca instead of Ubuntu 16.04.3,
```

Comment:
Could you please clarify where this (copied below) string comes from?

Reply:
I am sure it shall be derived from https://docs.openstack.org/image-guide/obtain-images.html#ubuntu though not sure how.

Reply:
One can find the ID by listing all private images:

```
$ openstack image list --private
+--------------------------------------+----------------+--------+
| ID                                   | Name           | Status |
+--------------------------------------+----------------+--------+
| 7fbe9a7c-05e3-4a80-a59f-682fef4d311e | Ubuntu 14.04.5 | active |
| d324da4b-b827-44fb-8331-bfad41a28aca | Ubuntu 16.04.3 | active |
+--------------------------------------+----------------+--------+
```

(one could also use the UI of course).